CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -953,60 +999,83 @@ public let DECL_NODES: [Node] = [
     kind: .functionParameterList,
     base: .syntaxCollection,
     nameForDiagnostics: "parameter list",
+    documentation: """
+      A list of function parameters that are type annotated and may have an optional label.
```

Comment:
The parameter labels aren’t optional, right? Or am I missing something?

Reply:
I meant specifying argument labels before parameter name, this is an optional action, is it?

Reply:
Did you mean
- That the argument label could be `_`, eg. `func foo(_ a: Int)`
  - In this case, I would still consider there to be an explicit label on the declaration side, it’s just that it’s `_`.
- That you can omit the internal name, eg. `func foo(a: Int)`
  - In this case the function still has an argument label `a`. It’s the internal or second name that you are omitting, which defines the name of the parameter inside the function’s body, which then defaults to the label

In either case, I would change the documentation to

```suggestion
      A list of function parameters that are type annotated and a label.
```

Reply:
You got it :)